### PR TITLE
Release 2.5.0

### PR DESCRIPTION
**Pull request by outofambit**
cc @niik @rafeca

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "2.5.0": [
+      "[New] Add, push, and view git tags in Desktop - #9424",
```
Comment (billygriffin) [inline changelog.json:4]: The intent here is just to increase discoverability a bit for people who want to use this but aren't sure where to look.
```suggestion
      "[New] Add, push, and view Git tags on commit history in Desktop - #9424",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+      "[Added] Offer option to choose file from one branch or the other when resolving merge conflicts - #9702",
+      "[Added] Add context menu for highlighted text in diff view - #5100. Thanks @HashimotoYT!",
+      "[Fixed] Retain default branch option in \"Create Branch\" dialog when user focuses away from Desktop - #9611",
+      "[Improved] Clean up copy for onboarding sign-in flow - #9715",
+      "[Improved] Emphasize sign in via browser option in Welcome flow - #9663",
+      "[Improved] Update verbage in repository publish error - #9646",
```
Comment (nerdneha) [inline changelog.json:10]:
```suggestion
      "[Improved] Update verbiage in repository publish error - #9646",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "2.5.0": [
+      "[New] Add, push, and view git tags in Desktop - #9424",
+      "[Added] Offer option to choose file from one branch or the other when resolving merge conflicts - #9702",
+      "[Added] Add context menu for highlighted text in diff view - #5100. Thanks @HashimotoYT!",
+      "[Fixed] Retain default branch option in \"Create Branch\" dialog when user focuses away from Desktop - #9611",
+      "[Improved] Clean up copy for onboarding sign-in flow - #9715",
+      "[Improved] Emphasize sign in via browser option in Welcome flow - #9663",
```
Comment (billygriffin) [inline changelog.json:9]: Just to provide a bit more context here
```suggestion
      "[Improved] Emphasize signing in to GitHub via browser due to username/password deprecation - #9663",
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "2.5.0": [
+      "[New] Add, push, and view git tags in Desktop - #9424",
+      "[Added] Offer option to choose file from one branch or the other when resolving merge conflicts - #9702",
+      "[Added] Add context menu for highlighted text in diff view - #5100. Thanks @HashimotoYT!",
+      "[Fixed] Retain default branch option in \"Create Branch\" dialog when user focuses away from Desktop - #9611",
+      "[Improved] Clean up copy for onboarding sign-in flow - #9715",
```
Comment (billygriffin) [inline changelog.json:8]: I don't think this made it to prod?
```suggestion
```

File: changelog.json [hunk 5]
```diff
@@ -1,5 +1,15 @@
...
+      "[Improved] Emphasize sign in via browser option in Welcome flow - #9663",
+      "[Improved] Update verbage in repository publish error - #9646",
+      "[Improved] Offer to retry cloning if it fails - #926. Thanks @Daniel-McCarthy!"
```
Comment (billygriffin) [inline changelog.json:11]:
```suggestion
      "[Improved] Offer to retry if cloning a repository fails - #926. Thanks @Daniel-McCarthy!"
```

File: changelog.json (hunk 2 above)
Comment (billygriffin) [inline changelog.json:10]:
```suggestion
      "[Improved] Improve error message when publishing a private repository fails - #9646",
```

File: changelog.json (hunk 3 above)
Comment (outofambit) [reply changelog.json:9]: hmmm i don't know if that deprecation is publicly announced yet @billygriffin. or has it?

File: changelog.json (hunk 4 above)
Comment (outofambit) [reply changelog.json:8]: @billygriffin these changes are a delta from the current wording on prod, so we should include this note.

File: changelog.json (hunk 3 above)
Comment (billygriffin) [reply changelog.json:9]: @outofambit It has been! https://developer.github.com/changes/2020-02-14-deprecating-oauth-auth-endpoint/

**Comment by outofambit** [comment]
its shipped!

**Pull request merged**